react/utils/client/links/versionSplitterLink.ts

```diff
@@ -187,7 +187,7 @@ const observableFromOperations = (operations: Operation[], forward: NextLink) =>
   }) as Observable<FetchResult<{ [key: string]: any }, Record<string, any>, Record<string, any>>>
 
 export const versionSplitterLink = new ApolloLink(
-  (operation: Operation, forward?: NextLink) => {
+  (operation: Operation, forward?: NextLink) : any => {
```

Comment:
I think this is can be deleted

Reply:
It throughs an error if I keep it
![image](https://user-images.githubusercontent.com/284515/60200829-47590280-981d-11e9-9fce-110690aaa608.png)